src/test/java/com/k3ntako/HTTPServer/ResponseTest.java

```diff
@@ -0,0 +1,37 @@
+package com.k3ntako.HTTPServer;
+
+import com.k3ntako.HTTPServer.mocks.RequestMock;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ResponseTest {
+
+  @Test
+  void createResponse() {
+    HashMap<String, String> headers = new HashMap<>();
+    headers.put("Content-Length", "17");
+    headers.put("Content-Type", "text/html; charset=UTF-8");
+
+
+    var request = new RequestMock(
```

Comment:
No need to create a mock here. You've set up your request class very well so you can craft a real request and send it in.